README.md

```diff
@@ -163,20 +163,31 @@ Run every 2 minutes
 
 https://mbtace.slack.com/files/U32MH8RCK/F9F0HTF96/apicalls.json
 
-## Docker
+## Docker in Dev
 
 Build tagged container:
 
 `docker build -t api-checker:latest .`
 
-To demo this in Docker using dev config, run this command after building:
 
-`docker run -e "API_CHECKER_CONFIGURATION=$(cat ./priv/dev_checks_config.json)" api-checker:latest`
+To demo this in Docker using dev config and using the cookie `a_super_secret_cookie` run this command after building:
+
+Note: the `ERLANG_COOKIE` env var should be secret and secure in production as it allows connection to the running erlang cluster which allows execution of arbitrary code.
+
+`docker run -e "API_CHECKER_CONFIGURATION=$(cat ./priv/dev_checks_config.json)" -e "ERLANG_COOKIE=a_super_secret_cookie" api-checker:latest`
```

Comment:
@paulswartz This is how to get the `ERLANG_COOKIE` into the docker container. Good call on the `REPLACE_OS_VARS=true`.